tests/driver_lis3mdl/main.c

```diff
@@ -21,11 +21,12 @@
 
 #include <stdio.h>
 
-#include "xtimer.h"
+#include "timex.h"
+#include "ztimer.h"
 #include "lis3mdl.h"
 #include "lis3mdl_params.h"
 
-#define SLEEP_USEC  (800 * 800U)
+#define SLEEP_MS  (640 * MS_PER_SEC)
```

Comment:
```suggestion
#define SLEEP_MS  (640)
```

Otherwise the duration would change

Reply:
Thanks! Stupid mistake...